src/DocsGenerator/Helpers/CategoryHelper.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using Codacy.Engine.Seed.Patterns;
+
+namespace Codacy.TSQLLint.DocsGenerator.Helpers
+{
+    /// <summary>
+    /// Helper to convert to codacy category from a TSQLLint rule
+    /// </summary>
+    public static class CategoryHelper
+    {
+        /// <summary>
+        /// This map a rule to a codacy category.
+        /// </summary>
+        /// <param name="rule">rule name</param>
+        /// <returns>Codacy category</returns>
+        /// <exception cref="NotImplementedException">When found a non explicit mapped rule</exception>
+        public static Category ToCategory(string rule)
+        {
+            switch (rule)
+            {
+                case "conditional-begin-end": return Category.ErrorProne;
+                case "cross-database-transaction": return Category.ErrorProne;
+                case "data-compression": return Category.Performance;
+                case "data-type-length": return Category.Performance;
+                case "disallow-cursors": return Category.Performance;
+                case "full-text": return Category.Performance;
+                case "information-schema": return Category.ErrorProne;
+                case "keyword-capitalization": return Category.CodeStyle;
+                case "linked-server": return Category.Performance;
+                case "multi-table-alias": return Category.ErrorProne;
+                case "named-constraint": return Category.ErrorProne;
+                case "non-sargable": return Category.Performance;
+                case "object-property": return Category.ErrorProne;
+                case "print-statement": return Category.ErrorProne; //FIXME: check if theres a better category

```

Comment:
It looks more likely to me that this could cause a security issue or not a being a best practice, rather than an error.

Reply:
Fixed. If you fing other missmatched category or level tell me, otherwise, I'll remove `FIXME` from the code.